front/tests/009-stdcm-between-two-trains.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { test, expect } from '@playwright/test';
+import { PlaywrightHomePage } from './pages/home-page-model';
+import PlaywrightRollingstockModalPage from './pages/rollingstock-modal-model';
+import { PlaywrightSTDCMPage } from './pages/stdcm-page-model';
+import PlaywrightMap from './pages/map-model';
+import VARIABLES from './assets/operationStudies/testVariables';
+import PATH_VARIABLES from './assets/operationStudies/testVariablesPaths';
+import PlaywrightScenarioPage from './pages/scenario-page-model';
+import { ProjectPage } from './pages/project-page-model';
+import { StudyPage } from './pages/study-page-model';
+
+const pathfindingDistance = VARIABLES.stdcm.pathfindingDistance
+  ? VARIABLES.stdcm.pathfindingDistance
+  : VARIABLES.pathfindingDistance;
+
+const rollingStockTestID = VARIABLES.stdcm.rollingstockTestID
+  ? VARIABLES.stdcm.rollingstockTestID
+  : VARIABLES.rollingstockTestID;
+
+const originSearch =
+  PATH_VARIABLES.stdcm && PATH_VARIABLES.stdcm.originSearchDijon
+    ? PATH_VARIABLES.stdcm.originSearchDijon
+    : PATH_VARIABLES.originSearch;
+
+const destinationSearch =
+  PATH_VARIABLES.stdcm && PATH_VARIABLES.stdcm.destinationSearchMacon
+    ? PATH_VARIABLES.stdcm.destinationSearchMacon
+    : PATH_VARIABLES.destinationSearch;
+
+test.describe('Testing if stdcm between two trains works well', () => {
+  let playwrightHomePage;
+  let playwrightSTDCMPage;
+  let scenarioPage;
+  let projectPage;
+  let studyPage;
```

Comment:
As these variables are not shared between the test and its before each, you can instantiate them directly in the test / beforeEach
```suggestion
```

Reply:
done